pyproject.toml

```diff
@@ -167,6 +167,7 @@ example-app-with-view-override = {path = "examples/example_app_with_view_overrid
 factory-boy = "~3.3.1"
 # Factory Boy dependency. Don't upgrade to v26.x until breaking change is fixed https://github.com/joke2k/faker/issues/2070
 faker = ">=0.7.0,<26.0.0"
+kubernetes = "31.0.0"
```

Comment:
Please add a brief comment documenting the purpose kubernetes serves in Nautobot development. :-)